packages/component-base/src/slot-controller.d.ts

```diff
@@ -25,5 +25,10 @@ export class SlotController implements ReactiveController {
    */
   node: HTMLElement;
 
+  /**
+   * Return a reference to the node managed by the controller.
+   */
+  getSlotChild(slotName: string): Node;
```

Comment:
The controller already knows the slot name so this function shouldn't accept an arbitrary slot name as a parameter?

Reply:
I wonder if this method is needed at all since we already store a reference to the node.

Reply:
The method is needed because we don't yet have `FlattenedNodesObserver` logic.
Once that is added, we probably can always rely on the `node` property.